xml/System.Threading/Thread.xml

```diff
@@ -1858,6 +1858,8 @@ Main thread: ThreadProc.Join has returned.  Press Enter to end program.
 
 This value is cached and periodically refreshed. It may be stale by the time it's used: consuming code must not rely on its accuracy for correctness.
 
+There is also no guarantee the result will be 0-based, and consuming code should not rely on that behavior.
```

Comment:
```suggestion
The value is not guaranteed to be zero-based processor number.
```

Reply:
I do not think we need to say that the code should not rely on something that is not guaranteed.

Reply:
Works for me.